requirements.txt

```diff
@@ -1,5 +1,4 @@
-arrow==0.12.1
-click==6.7
+click==7.0
 coloredlogs==10.0
 docutils==0.14
 future==0.16.0
```

Comment:
do we need to add unicodecsv

Reply:
oh huh, it was already imported by the bulkdata module so I assumed it was here, but it must be getting pulled in transitively. yeah we should pin it